### PR TITLE
Add api

### PR DESCRIPTION
**Pull request by lukman155**
Build an API that is RESTful.
Understand importance of APIs and docs following standards.

### CODE REVIEW

File: app/controllers/api/comments_contrroler.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Api::CommentsController < ApplicationController
+  def index
+    @comments = Post.find(params[:post_id]).comments
+    redirect_to do |format|
+      format.json { render json: @comments }
+    end
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
```
Comment (mckent05) [inline app/controllers/api/comments_contrroler.rb]: - You have done a good job so far, but please note that this endpoint should create a comment for a particular post, so it is necessary to specify the post for which the comment is created and also the user creating the comment i.e establishing the `belongs-to` relationship. Please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @lukman155, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job creating endpoints for your project 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/api/comments_contrroler.rb (hunk 1 above)
Comment (rloterh) [reply app/controllers/api/comments_contrroler.rb]: resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi  Girma & Lukman,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Good job implementing recommended changes!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

**You can also consider:**
- Kindly endeavor to remove Live demo link this app is not yet deployed.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**